morgoth/addons/models.py

```diff
@@ -78,12 +78,16 @@ def __repr__(self):
         return '<VersionNumber: {}>'.format(self)
 
     def __eq__(self, other):
-        if not isinstance(other, VersionNumber):
+        if other is None:
```

Comment:
DRF appears to do some comparison when `PATCH`ing to see if the field has changed.